python/llm/example/NPU/HF-Transformers-AutoModels/Multimodal/README.md

```diff
@@ -32,6 +33,9 @@ pip install torchvision
 # [optional] for MiniCPM-V-2_6
 pip install timm torch==2.1.2 torchvision==0.16.2
 
+# [optional] for Bce-Embedding-Base-V1
```

Comment:
with ipex-llm[npu] installation option, transformers==4.40.0 is installed, no need here.